include/cuco/detail/bloom_filter/arrow_filter_policy.cuh

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cuco/hash_functions.cuh>
+
+#include <cuda/std/__algorithm_>
```

Comment:
```suggestion
```
`cuda/std/__algorithm_` is still an internal experimental feature. Let's just use public headers for safety: `cuda::std::min/max` can be accessed from `<cuda/std/functional>` which is already included.

Reply:
I actually tried that but I keep getting a compiler error: `/home/coder/cuCollections/include/cuco/detail/bloom_filter/arrow_filter_policy.cuh(111): error: namespace "cuda::std" has no member "min"`

Reply:
Thank you for looking into this. Looks like the CCCL hasn't established a consistent accessibility solution for this utility yet.